kubernetes/nodecollection_test.go

```diff
@@ -541,7 +541,7 @@ func TestDownloadNodeData(t *testing.T) {
 			t.Error("Expected error for nodename \"proxyNode\"")
 		}
 
-		if errFromList.Error() != "Provider ID for node does not exist. "+
+		if errFromList.Error() != "provider ID for node does not exist. "+
```

Comment:
I see this string is used twice (once in the code/once in this test) - do you think it's worth creating a variable?

Reply:
I think with just two uses and being so short, it's fine as is.